binding.gyp

```diff
@@ -1,8 +1,28 @@
 {
-	"targets": [ {
-		"target_name":  "journal_send",
-		"sources":      [ "src/journal_send.cc" ],
-		"include_dirs": [ "<!(node -e \"require('nan')\")" ],
-		"libraries":    [ "<!@(pkg-config --libs-only-l --silence-errors libsystemd || pkg-config --libs-only-l libsystemd-journal)" ]
-	} ]
+	"targets": [
+		{
+			"target_name": "journal_send",
+			"sources": [
+				"src/journal_send.cc"
+			],
+			"libraries": [
+        "-lsystemd",
+      ],
+			'cflags!': [ '-fno-exceptions' ],
+      'cflags_cc!': [ '-fno-exceptions' ],
+      'include_dirs': ["<!@(node -p \"require('node-addon-api').include\")"],
+      'dependencies': ["<!(node -p \"require('node-addon-api').gyp\")"],
+      'defines': [ 'NAPI_DISABLE_CPP_EXCEPTIONS' ],
+      'conditions': [
+        [
+					'OS=="mac"',
+          {
+            'xcode_settings': {
+              'MACOSX_DEPLOYMENT_TARGET': '10.9',
+            },
+          }
+        ]
+			]
```

Comment:
```suggestion
```

Systemd is Linux-only. This won't compile under MacOS

Reply:
Actually this will compile on MacOS but will NOT run on MacOS. I think I'll just add restriction in `package.json` to mark it only for `linux` so Mac users can add it as `optional` dependency.

Reply:
but you are right, I'll remove this section just to avoid any confusion

Reply:
I think the restriction is already in the `package.json`, right?
You should be able to use this module as optional dependency. Windows and MacOS will just ignore this dependency.